src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykAuthPanel.kt

```diff
@@ -29,18 +28,20 @@ import snyk.analytics.AuthenticateButtonIsClicked
 import snyk.analytics.AuthenticateButtonIsClicked.EventSource
 import snyk.analytics.AuthenticateButtonIsClicked.Ide
 import snyk.analytics.AuthenticateButtonIsClicked.builder
-import java.awt.Insets
 import java.awt.event.ActionEvent
 import javax.swing.AbstractAction
 import javax.swing.JButton
 import javax.swing.JLabel
 import javax.swing.JPanel
 
 class SnykAuthPanel(val project: Project) : JPanel(), Disposable {
+    companion object {
+        const val AUTHENTICATE_BUTTON_TEXT = "Test code now"
+    }
```

Comment:
minor: Companion object should be in the end by [convention](https://kotlinlang.org/docs/coding-conventions.html#source-file-organization) we currently follow (but [that's discussable](https://discuss.kotlinlang.org/t/companion-objects-on-bottom/17797))